validate_email/validate_email.py

```diff
@@ -63,12 +68,8 @@ def validate_email(email_address: str, *args, **kwargs):
     try:
         return validate_email_or_fail(email_address, *args, **kwargs)
     except SMTPTemporaryError as error:
-        message = f'Validation for {email_address!r} ambigious: {error}'
-        if kwargs.get('debug'):
-            LOGGER.warning(msg=message)
+        LOGGER.info(msg=f'Validation for {email_address!r} ambigious: {error}')
```

Comment:
I'd keep the warnings in the case of the debug flag set. Users were complaining about them being unable to turn off logging before.

Reply:
See #30

Reply:
Just to make sure: have you seen my comment in commit a0f1cd1b046cb76aef052246db5f4ce750c00f86, and do you disagree with it? In short, my reasoning for the change was:
* Info fits better than warning, because it's expected library behaviour in a standard usecase
* Info is not output by default, so everybody not configuring logging explicitly will get no logging output
* It makes it possible to enable our logging messages without enabling the very verbose and non-configurable smtplib debug output.

I have no problem reverting this change, but I want to make sure you really disagree with my reasons.

Reply:
You have a point there. I remember somebody complaning seeing log messages coming from this module when he used it with a command line tool he built. I can't find the issue now, but I remember having it posted. That said AFAIK `info` doesn't get logged to the stderr so I'll mark this as resolved and leave it as is.